templates/transport.md

```diff
@@ -0,0 +1,62 @@
+# Transport Name
+
+Describe tranport at a high level here.
+
+# Design and Specification
+
+Add links to specifications and design docs.
+
+# Presentation API functionality
+
+Describe how the transport meets [functional requirements for the Presentation
+API](../requirements.md#req-p2-initiation), especially requirements 2-6.
+
+# Remote Playback API functionality
```

Comment:
A minor note that we should update this when the Remote Playback API requirements have been added.